packages/react-divider/src/components/Divider/renderDivider.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-compose/lib/next/index';
+import { DividerState } from './Divider.types';
+import { dividerShorthandProps } from './useDivider';
+
+/**
+ * Redefine the render function to add slots. Reuse the divider structure but add
+ * slots to children.
+ */
+export const renderDivider = (state: DividerState) => {
+  const { slots, slotProps } = getSlots(state, dividerShorthandProps);
+  const { children } = state;
+
+  return slotProps.vertical ? renderVertical(slots, slotProps, children) : renderHorizontal(slots, slotProps, children);
+  /*
+  return (
+    <slots.root {...slotProps.root}>
+      {loading && <slots.loader {...slotProps.loader} />}
+      {iconPosition !== 'after' && <slots.icon {...slotProps.icon} />}
+      {!iconOnly && <slots.content {...slotProps.content} />}
+      {iconPosition === 'after' && <slots.icon {...slotProps.icon} />}
+    </slots.root>
+*/
+};
+
+const renderHorizontal = (slots: Record<string, any>, slotProps: Record<string, any>, children: any) => {
+  return children ? (
+    <slots.root {...slotProps.root}>
+      <div>{children}</div>
+    </slots.root>
+  ) : (
+    <slots.root {...slotProps.root} />
+  );
+};
+
+const renderVertical = (slots: Record<string, any>, slotProps: Record<string, any>, children: any) => {
+  return children ? (
+    <slots.root {...slotProps.root}>
+      <div>{children}</div>
+    </slots.root>
+  ) : (
+    <slots.root {...slotProps.root} />
+  );
+};
```

Comment:
Seems like these 2 are exactly the same so I think this should be condensed into one unique `renderDivider` function.

Reply:
no longer applicable